awssecretssync/awssync.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+import json
+import os
+import sh
+
+from awssecretssync.passeditor import PASS_CONTENT_FILE
+
+
+def _save_to_passwordstore(secret_name, value):
```

Comment:
@huogerac The pass has a special care with the persistence of the passwords.

> Insert a new password or edit an existing password using the default text editor specified by the environment variable EDITOR or using editor(1) as a fallback.  This  mode makes  use  of  temporary  files for editing, but care is taken to ensure that temporary files are created in /dev/shm in order to avoid writing to difficult-to-erase disk sectors. If /dev/shm is not accessible, fallback to the ordinary TMPDIR location, and print a warning.

It might not be a good idea to persist the password in a file, and a file that anyone can read...

Reply:
awsome!
https://github.com/spacetimelabs/aws-secrets-sync/issues/2